### PR TITLE
Benchmark the compiler with sbt phases enabled

### PR DESCRIPTION
**Pull request by allanrenucci**

### CODE REVIEW

File: profiles/default.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ charts:
       lines:
         - key:  dotty
           label: bootstrapped
+        - key:  dotty-sbt
+          label: with sbt phases
```
Comment (allanrenucci) [inline profiles/default.yml:8]: Is it OK to have a label with spaces?

Comment (liufengyun) [reply profiles/default.yml:8]: Yes, it's OK, what this is assumed to test?

Comment (allanrenucci) [reply profiles/default.yml:8]: This is meant to test the performance impact when we enable incremental compilation

Comment (liufengyun) [reply profiles/default.yml:8]: BTW, this will make the default test time from 2h to 3h. Maybe we only enable it for merges?

Comment (allanrenucci) [reply profiles/default.yml:8]: Yes, only for merge is fine as long as I can request this benchmark on a PR. New profile?

Comment (liufengyun) [reply profiles/default.yml:8]: Yes, I think we need a profile for master (`master.yml`), and a specific profile for this one (`sbt.yml`), as usually people may not want it.

File: profiles/default.yml [hunk 2]
```diff
@@ -131,6 +133,9 @@ scripts:
     dotty:
         - measure $(find $PROG_HOME/dotty/compiler/src/dotty -name *.scala -o -name *.java)
 
+    dotty-sbt:
+        - measure -Yforce-sbt-phases $(find $PROG_HOME/dotty/compiler/src/dotty -name *.scala -o -name *.java)
+
```
Comment (liufengyun) [inline profiles/default.yml:138]: @allanrenucci Just to confirm, testing dotty is extremely expensive (1 hour), it seems we can also test other projects with the same option.

**Review by liufengyun — COMMENTED** [review]
Do we need to collect history data points?

**Comment by allanrenucci** [comment]
> Do we need to collect history data points?

Can we collect for the last two weeks?

**Comment by liufengyun** [comment]
Sure, I'll do that once it's merged.

**Pull request merged**